__tests__/app.test.js

```diff
@@ -43,4 +43,40 @@ describe("GET /api", () => {
             })
         })
     })
+    describe("/articles", () => {
+        describe("/:article_id", () => {
+            test("200: Should retrieve an article by id", () => {
+                return request(app)
+                .get('/api/articles/1')
+                .expect(200)
+                .then((res) => {
+                    expect(res.body.article).toHaveProperty("author", expect.any(String));
+                    expect(res.body.article).toHaveProperty("title", expect.any(String));
+                    expect(res.body.article).toHaveProperty("article_id", expect.any(Number));
```

Comment:
we can expect this article_id to be 1 here :)

Reply:
This has been refactored.